scripts/new-version-pr-generator.js

```diff
@@ -90,57 +90,65 @@ The automatic branch creation for ${repository} failed. This can be due to many
 
 mkdir(sandbox);
 
-repositories.forEach((repository) => {
-  try {
-    const repositoryDirectory = resolve(sandbox, repository);
-
-    // Clone into the sandbox directory
-    retry(
-      execSync(
-        `git clone --branch ${baseBranch} --single-branch https://${polarisBotToken}@github.com/Shopify/${repository}.git`,
-        {
-          cwd: sandbox,
-          stdio: 'inherit',
-        },
-      ),
-    );
-
-    const commands = [
-      `git checkout -b update-polaris-${releaseVersion}`,
-      `npx yarn@${YARN_VERSION} upgrade @shopify/polaris@${releaseVersion.replace(
-        'v',
-        '',
-      )} --no-progress --ignore-engines`,
-      `git add package.json yarn.lock`,
-      `${shopifyPolarisBotGitOverride} git commit -m "Update @shopify/polaris to ${releaseVersion}" --author "${polarisBotName} <${polarisBotEmail}>" -m "${releaseVersion}" --allow-empty`,
-      `git push origin update-polaris-${releaseVersion}`,
-      `curl -d '${JSON.stringify({
-        title: `Update @shopify/polaris to ${releaseVersion} 🚀`,
-        body: pullRequestTemplate(repository, releaseVersion),
-        head: `update-polaris-${releaseVersion}`,
-        base: baseBranch,
-      })}' -X POST https://api.github.com/repos/shopify/${repository}/pulls?access_token=${polarisBotToken}`,
-    ];
-
-    // Run the commands in the cloned repository directories
-    commands.forEach((command) => {
-      retry(
-        execSync(command, {
-          cwd: repositoryDirectory,
-          stdio: 'inherit',
-        }),
+const jobs = repositories.map((repository) => {
+  return new Promise(async (resolve, reject) => {
```

Comment:
You can simplify this as:
```js
const jobs = repositories.map((repository) => (
  new Promise(async (resolve, reject) => {
```

Reply:
I tried doing this but the linter is not liking the end `);`

Reply:
oh I didn't realize we had a linting rule that blocked this